src/app/spaces/space.service.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable, Inject } from '@angular/core';
 import { Headers, Http, URLSearchParams } from '@angular/http';
 import { cloneDeep } from 'lodash';
 import { AuthenticationService, Logger, User, UserService } from 'ngx-login-client';
-import { Observable } from "rxjs";
+import { Observable } from 'rxjs';
```

Comment:
While this works, it is better to be more specific.

`import { Observable } from 'rxjs/Observable';`